src/eggd_sompy.sh

```
@@ -31,13 +31,36 @@ main() {
 
             prefix=$(basename $(basename ${query_vcf%%_*}))
 
+            # normalise query VCF
+            query_vcf_name=$(sed -E 's/\.vcf(\.gz)?//g' <<< $(basename $query_vcf))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)